src/main/java/LottoSimulator.java

```diff
@@ -0,0 +1,43 @@
+public class LottoSimulator {
+    public LottoSimulator(){
+        while(true){
+            Output.startScreen();
+            checkInputNumber(Input.getInputValue());
+        }
+    }
+
+    public void checkInputNumber(int inputValue){
+        if(inputValue == 1) {
+            Simulator simulator = new Simulator();
+        } else {
+            Output.ExitScreen();
+        }
```

Comment:
```suggestion
        if(inputValue != 1) {
            Output.ExitScreen();
        }
```

Simulator simulator = new Simulator(); 별도로 실행
유효성 검사 로직과 같이 있을 이유가

Reply:
```java
if(Input.getInputValue() != 1){
   Output.ExitScreen();
 }
RandomNumber randomNumber = new RandomNumber(Input.getSimulatorInput());
```
`ExitScreen()`안에 `System.exit(0);` 코드가 있어 검사 이후 실행되도록 변경했습니다!